modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainerProvider.java

```diff
@@ -6,7 +6,8 @@
 public class PostgreSQLContainerProvider extends JdbcDatabaseContainerProvider {
     @Override
     public boolean supports(String databaseType) {
-        return databaseType.equals(PostgreSQLContainer.NAME);
+        return databaseType.equals(PostgreSQLContainer.NAME)
+            || databaseType.equals(PostgisContainer.NAME);
```

Comment:
This would AFAIK lead to the creation of a `PostgreSQLContainer` when `postgis` is used in the JDBC url